proto/config/appconfig.proto

```diff
@@ -210,10 +208,6 @@ message AppInstanceConfig {
   // PatchEnvelopeRef is used to track all patch envelopes used by the app instance
   PatchEnvelopeRef patchRef = 23;
 
-  // allow AppInstance to discover other AppInstances
```

Comment:
But this was already removed in #52 based on the diffs, right?

Reply:
Yes, but #46 introduced this field once again with different number, if you look at master [here](https://github.com/lf-edge/eve-api/blob/main/proto/config/appconfig.proto#L215), you'll see this field

Reply:
Ah - presumably due to some mismerge. Got it.